woocommerce/woocommerce
# Remove unneeded files from distribution package

### All Submissions:






### Changes proposed in this Pull Request:



This PR just does some house cleaning to remove unneeded files from the distribution package.

### How to test the changes in this Pull Request:

1. cd `plugins/woocommerce-beta-tester`
2. Run `pnpm run build:zip`
3. Unzip the package and ensure all essential files are included and that it can activate on your local site without errors.

### Other information:




### FOR PR REVIEWER ONLY:
